apps/reference/docs/guides/auth/auth-security.mdx

```diff
@@ -0,0 +1,458 @@
+not store the token issued by the provider in any way, but does make it
+available to your browser's app should you need further access to it. Each
+subsequent sign-in with OAuth updates the user's identifying information in
+Suapabse Auth.
+
+### Automatic linking
+
+Users with the same email address in different social login providers will get
+one single user account in Supabase Auth. We believe this to be the most common
+expected behavior for modern applications.
+
+:::warning
+**Not all social login providers can be trusted!** Some providers will allow
+social login with your application **even if the user's email address is not
+confirmed.** This has the unintended side-effect that attackers could abuse
+such a social login provider to gain access to a legitimate user's account.
+**Always make sure you verify the behavior of social login providers with
+regards to confirming the user's email address!**
+:::
+
+### Magic links
+
+Supabase Auth provides passwordless sign-in with magic links. Magic links are a
+URL that contains a single-use and unguessable code. By default this URL is
+hosted by the Supabase Auth server and immediately redirects back to your app.
+
+You should be aware that email messages are inherently insecure, although the
+default SMTP settings from Supabase Auth always attempt to deliver email
+messages via TLS to the receiving server.
+
+There are a number of email clients, enterprise and end-user security software,
+which inspect email messages for malware. Often this software will visit links
+in messages (sometimes multiple times) in an attempt to scan for viruses.
+Unfortunately this has the effect of signing up a user without their knowledge
+or express consent. Furthermore, it does give out an access and refresh token
+to this software which could be used maliciously.
+
+:::warning
+We do not recommend using magic-link sign in without customization for
+applications requiring a high level of authentication security. Should you wish
+to still use magic-links you can prevent this type of issues by modifying the
+magic link email template to point to your application where you can present
+the user with a button that will lead to the magic link URL.
+:::
+
+## User information
+
+Supabase Auth stores user account information in your project's database. This
+information is provided by your application or collected from third-party
+identity providers.
+
+Email addresses, phone numbers and other personally identifiable information is
+not stored encrypted in your project's database. If you need to store encrypted
+data for users, we recommend you check out Supabase Vault as an easy and safe
+way to achieve this.
+
+### User enumeration protection
+
+User enumeration is a technique where third-parties gather information about
+the existence of accounts on a system. In and of itself the use of this
+technique does not indicate malicious behavior, but it may be used as part of
+such an effort.
```

Comment:
```suggestion
User enumeration is a technique where third-parties gather information about
the existence of accounts on a system. The use of this
technique does not indicate malicious behavior, but it may be used as part of
such an effort.
```